apis/Google.Cloud.Spanner.V1/Google.Cloud.Spanner.V1/SpannerClientBuilder.cs

```diff
@@ -46,10 +46,9 @@ public new EmulatorDetection EmulatorDetection
         /// May return a builder that will connect to the emulator under certain conditions.
         /// </summary>
         /// <remarks>
-        /// Returns null if <see cref="EmulatorDetection"/> is <see cref="EmulatorDetection.None"/>
-        /// or <see cref="EmulatorDetection.ProductionOnly"/>. Otherwise, returns a builder to
-        /// connect to the emulator using the endpoint specified by the emulator host environment
-        /// variable, and insecure credentials.
+        /// It returns null if the emulator should not be used, i.e. if <see cref="EmulatorDetection"/> is
```

Comment:
Nit: change the `i.e.` to `e.g.`, as there are other cases where the emulator shouldn't be used. But otherwise, this is fine, thanks.